data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -25,6 +25,17 @@ do
     sudo podman inspect "${container}" >& "${ARTIFACTS}/bootstrap/pods/${container}.inspect"
 done
 
+echo "Gathering rendered assets..."
+mkdir -p "${ARTIFACTS}/rendered-assets"
+cp -r /var/opt/openshift/ "${ARTIFACTS}/rendered-assets"
+# remove sensitive information
+# TODO leave tls.crt inside of secret yaml files
+find "${ARTIFACTS}/rendered-assets" -name "*secret*" -print0 | xargs -0 rm
```

Comment:
nit: you can drop `xargs` and the pipe with: `-print0 | xargs -0 rm` -> `-execdir rm {} \+`

Reply:
> nit: you can drop `xargs` and the pipe with: `-print0 | xargs -0 rm` -> `-execdir rm {} \+`

All bash sucks.  `xargs` is tremendously faster with larger lists.  I like consistency.

Reply:
> `xargs` is tremendously faster with larger lists.

`find` piping to `xargs` is faster than `find` calling `rm` directly?  How long a list do you need to have before this effect shows up?

```console
$ for x in $(seq 1000); do for y in $(seq 100); do touch $x.$y; done; done
$ time find . -type f -execdir rm -f {} \+

real	0m0.967s
user	0m0.155s
sys	0m0.801s
$ for x in $(seq 1000); do for y in $(seq 100); do touch $x.$y; done; done
$ time (find . -type f -print0 | xargs -0 rm)

real	0m0.864s
user	0m0.141s
sys	0m0.761s
```

Reply:
> `find` piping to `xargs` is faster than `find` calling `rm` directly? How long a list do you need to have before this effect shows up?

Looks like your example exposes it as faster.  I would go with the structurally sound choice.

Reply:
Looks like a detail and taste. Both xargs and `-exec rm` look fine. Would leave it as it is and spend no energy here.